### ISSUE TITLE
Is there a download for the console plugin.

### ISSUE THREAD

**Issue by mikemast01** [vmware/webcommander#137]

**Comment by mikemast01** [comment]
Other question is using the vmware standalone remote console program for this? Where the standalone console program can be called from the powershell results.

**Comment by 9whirls** [comment]
https://my.vmware.com/web/vmware/details?downloadGroup=VMRC800&productId=491

**Comment by mikemast01** [comment]
Yes I downloaded that... I have not tried it yet, will webcommander call that instead of the web browser.?

**Comment by mikemast01** [comment]
Ok so would need to change code some for it to call the VMRC standalone instead of the web browser. Or make a c second command, leave the web browser one and make another that calles the standalone VMRC program.
HMM......

**Comment by 9whirls** [comment]
Do you mean to call VMRC on client machine( where you use browser to access webcommander) ?

**Comment by mikemast01** [comment]
Yes.

Mike



Sent from my Verizon Wireless 4G LTE smartphone


-------- Original message --------

**Comment by 9whirls** [comment]
Well, webcommander is pure html + javascript on client side. So it could not launch any program outside browser.

**Comment by mikemast01** [comment]
From what I was reading you are suppose to call it from a browser..  when I get back to office I will show you the link I was reading.

Mike



Sent from my Verizon Wireless 4G LTE smartphone


-------- Original message --------

**Comment by mikemast01** [comment]
ok you have to download the above link to get the plugin, and it also installs a standalone consol on your client. and if you go to c:\programfiles (x86)\VMware\VMware remote Console\   and run vmrc.exe it gives you this message.
![image](https://cloud.githubusercontent.com/assets/17012705/13472425/77d18fda-e07a-11e5-9f5f-39fddb0587af.png)

Doing some reading on this site.  ( great site btw)
http://www.virtuallyghetto.com/2014/10/standalone-vmrc-vm-remote-console-re-introduced-in-vsphere-5-5-update-2b.html

it looks like you can call this in a browser URL...
https://communities.vmware.com/thread/512410?start=0&tstart=0


Granted I have not tried yet, but it sounded like it could be done.

**Comment by mikemast01** [comment]
![image](https://cloud.githubusercontent.com/assets/17012705/13472591/34f60848-e07b-11e5-8c77-113b5f6cdc36.png)


He shows you can get the link from the web interface

**Comment by 9whirls** [comment]
That's not HTML, but Flex I believe.

**Comment by mikemast01** [comment]
Aww man..  I had my hopes up.  So to install the plugin , you have to install the VVMRC installer then.

**Comment by vmikemast** [comment]
looks like the latest update to webcommander now calls the VMRC console , instead of the web browser console.. so looks like being able to launch the console in webcommander now works...  YAY !
